src/shared/components/app/theme.tsx

```diff
@@ -20,22 +21,11 @@ export class Theme extends Component<Props> {
             href={`/static/assets/css/themes/${user.local_user_view.local_user.theme}.min.css`}
           />
         ) : (
-          [
-            <link
-              rel="stylesheet"
-              type="text/css"
-              href="/static/assets/css/themes/litely.min.css"
-              id="default-light"
-              media="(prefers-color-scheme: light)"
-            />,
-            <link
-              rel="stylesheet"
-              type="text/css"
-              href="/static/assets/css/themes/darkly.min.css"
-              id="default-dark"
-              media="(prefers-color-scheme: no-preference), (prefers-color-scheme: dark)"
-            />,
-          ]
+          <link
+            rel="stylesheet"
+            type="text/css"
+            href={`/static/assets/css/themes/${this.props.site.default_theme}.min.css`}
+          />
```

Comment:
iirc the default theme should be "browser", which isn't actually a theme, but uses either litely or darkly. So this should be checking for that, not removing the prefers color scheme stuff.